pkg/logs/tailers/container/tailer.go

```diff
@@ -388,7 +388,10 @@ func (t *Tailer) forwardMessages() {
 			tags = append(tags, t.tagProvider.GetTags()...)
 			origin.SetTags(tags)
 			// XXX(remy): is it OK recreating a message here?
-			t.outputChan <- message.NewMessage(output.GetContent(), origin, output.Status, output.IngestionTimestamp)
+			msg := message.NewMessage(output.GetContent(), origin, output.Status, output.IngestionTimestamp)
+			// Preserve ParsingExtra information from decoder output (including IsTruncated flag)
+			msg.ParsingExtra = output.ParsingExtra
+			t.outputChan <- msg
```

Comment:
I'm a bit concerned we'll forget to copy this info sometimes where it is needed. I wonder if (per the `XXX` comment) if we even need to create a new message or if we can just reuse it 🤔